### DISCUSSION TITLE
Text encoders

### DISCUSSION THREAD

**Discussion by Arwa-97** [keras-team/keras-cv#2148] [Q&A]
Greeting,
I wonder if in the fine tuned stable diffusion there's another text encoder 
that I can use other than CLIP with larger max prompt length?
If there's please mention documentation link/tutorial 

Thanks!

**Comment by divyashreepathihalli** [comment]
you are looking for a text encoder that is not CLIP for the stable diffusion model?

**Reply by Arwa-97** [reply]
yes I am, if it's possible to do

**Comment by dugujiujian1999** [comment]
1. Here is the method to get a larger prompt length. [long-prompt-weighting-stable-diffusion](https://github.com/huggingface/diffusers/tree/main/examples/community#long-prompt-weighting-stable-diffusion)
 2. Another text encoder (maybe): https://github.com/salesforce/LAVIS/tree/main/projects/blip-diffusion
